content/faq/serverless.md

```diff
@@ -209,36 +223,53 @@ functions:
 
 > info **Hint** To learn more about the Serverless framework, visit the [official documentation](https://www.serverless.com/framework/docs/).
 
-With this place, we can now navigate to the `main.ts` file and update our bootstrap code with the required boilerplate:
+With that in place, we can now create a file called `lambda.ts` inside `src` and place the necessary boilerplate:
 
 ```typescript
 import { NestFactory } from '@nestjs/core';
-import serverlessExpress from '@vendia/serverless-express';
-import { Callback, Context, Handler } from 'aws-lambda';
+import { default as serverlessExpress } from '@vendia/serverless-express';
```

Comment:
Why not keep the `default` import?

Reply:
Maybe I configured it wrong in the first tests and it didn't work, now I tested it again and it works fine, I will revert. to just `serverlessExpress`.